src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int maxColorValue = 5;
+    private static final int maxRandomValue = 10;
+    private static final String defaultColor = String.valueOf(ColorSupplier.Color.WHITE);
+    private static final int defaultRadius = 10;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomNumber = random.nextInt(maxColorValue);
+
+        switch (randomNumber) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            default:
+                return getRandomSquare();
+        }
+    }
+
+    public Figure getRandomCircle() {
```

Comment:
all these methods should be private
```suggestion
    private Figure getRandomCircle() {
```